ddtrace/internal/runtime/runtime_metrics.py

```diff
@@ -68,6 +69,11 @@ class RuntimeMetrics(RuntimeCollectorsIterable):
 
 
 def _get_interval_or_default():
+    if "DD_RUNTIME_METRICS_INTERVAL" in os.environ:
+        deprecate(
+            "`DD_RUNTIME_METRICS_INTERVAL` is deprecated and will be removed in a future version.",
+            removal_version="4.0",
```

Comment:
```suggestion
            removal_version="4.0.0",
```
Consistency nit, since we already have the full version here: https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/opentracer/__init__.py#L11